test/imageDims.test.js

```diff
@@ -1,24 +1,43 @@
-var test = require('tape');
-
+import test from 'tape';
+import nock from 'nock';
 import { parse } from '../src/ampl.js';
 
 test('image dimensions found', t=> {
-  var testMd =
-    '![image](https://cloud.githubusercontent.com/assets/12845233/12449931/68f11832-bf78-11e5-87ff-d34e6a3487c6.png)' +
-    '![image](http://www.lightningsafety.noaa.gov/photos/Lightning%202a.jpg)' +
-    '![image](http://www.notaurl.jpg)' +
-    '![image](https://raw.github.com/nelsonic/learning-istanbul/master/screenshots/97-percent-hides-malicious-code.png)';
+  const testMd =
+    `
+# h1
+
+![image](https://mock1.io/pic1.jpg)
+![image](http://www.mock2.io/pic2.jpg)
+![image](http://www.mock2.io/pic3.jpg)
+![image](http://www.mock2.io/notAPic)
+![image](http://notaurl.jpg)`;
+
+  nock('https://mock1.io')
```

Comment:
I don't see the reason for *needing* to *mock* the images?
why not create *real* tests with actual images loaded from imgur?
http://i.imgur.com/OcSUnWs.jpg
which will actually test the functionality...